src/sentry/static/sentry/app/views/performance/landing/durationHistogramSlider.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import {getTrackBackground, Range} from 'react-range';
+import {css} from '@emotion/core';
+
+import theme from 'app/utils/theme';
+
+const STEP_DEFAULT = 1;
+const MIN_DEFAULT = 0;
+const MAX_DEFAULT = 100;
+
+type Props = {
+  step?: number;
+  min?: number;
+  max?: number;
+  minValue: number;
+  maxValue: number;
+  width: number;
+  onMinChange: (val: number) => void;
+  onMaxChange: (val: number) => void;
+  onFinalChange: (values: number[]) => void;
+};
+
+export function getHistogramColors() {
+  const palette = theme.charts.getColorPalette(1);
+  return {
+    primary: palette[0],
+    highlight: theme.purple200,
+  };
+}
+
+/**
+ * This component is using react-range to provide a multi-thumb slider as this pattern may be temporary.
+ * If we move this to our standard components we should re-evaluate using our own or another lib.
+ */
+export function DurationHistogramSlider(props: Props) {
+  const primaryChartColor = getHistogramColors().primary;
+  const values = [props.minValue, props.maxValue];
+
+  const rangeMin = props.min || MIN_DEFAULT;
+  const rangeMax = props.max || MAX_DEFAULT;
```

Comment:
Can `props.max` somehow be `0`?

Reply:
it should, I should probable coalesce these instead with `null`